OpenRA.Mods.Common/Traits/World/Locomotor.cs

```diff
@@ -27,7 +27,8 @@ public enum CellFlag : byte
 		HasStationaryActor = 2,
 		HasMovableActor = 4,
 		HasCrushableActor = 8,
-		HasTemporaryBlocker = 16
+		HasTemporaryBlocker = 16,
+		HasHotActor = 32,
```

Comment:
Related to my earlier comments: how about `HasPassableNoStoppingActor`?

Reply:
Renamed.